<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicPlayer</class>
 <widget class="QWidget" name="MusicPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MusicPlayer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MusicPlayer{

}
</string>
  </property>
  <widget class="QFrame" name="menu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>76</y>
     <width>211</width>
     <height>505</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#menu{
background-color: #FFFFFF;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="likeListBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>191</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>我喜欢</string>
    </property>
    <property name="shortcut">
     <string>Alt+3</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="localListBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>191</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>本地音乐</string>
    </property>
    <property name="shortcut">
     <string>Alt+4</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="searchBtn_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>81</y>
      <width>191</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>搜索</string>
    </property>
    <property name="shortcut">
     <string>Alt+2</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="firstWidgetBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>191</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>推荐</string>
    </property>
    <property name="shortcut">
     <string>Alt+1</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>1071</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="searchWidget">
    <widget class="QLineEdit" name="searchText">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>18</y>
       <width>431</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">#searchText{
	border-radius: 20px;
	background-color: rgba(102, 190, 252, 0.5);
	padding-left: 20px;
	padding-right: 20px;

}</string>
     </property>
     <property name="placeholderText">
      <string>桜川 magic of live</string>
     </property>
    </widget>
    <widget class="QPushButton" name="searchBtn">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>23</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;在网络中搜索&lt;/p&gt;&lt;span&gt;（Enter）&lt;/span&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="shortcut">
      <string>Return</string>
     </property>
    </widget>
    <widget class="myListWidget" name="searchResultList">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>1071</width>
       <height>431</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget{color:rgb(54,54,54); background:rgb(255,255,255);border:1px solid gray;outline:0px;}
QListWidget::Item{height:30px;border:0px solid gray;padding-left:15;}
QListWidget::Item:hover{background:transparent;border:0px solid gray;padding-left:25;background:rgb(240, 240, 240);}
QListWidget::Item:selected{border-image:url(images/listwidget_h.png);border:0px solid gray;color:#000;}
QListWidget::Item:selected:active{background:rgb(229, 229, 229);border-width:0;}
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="downloadPath">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>20</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <underline>true</underline>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">#downloadPath{
	border: 0px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="likeListWidget">
    <widget class="myListWidget" name="likeList">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>1071</width>
       <height>431</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget{color:rgb(54,54,54); background:rgb(255,255,255);border:1px solid gray;outline:0px;}
QListWidget::Item{height:30px;border:0px solid gray;padding-left:15;}
QListWidget::Item:hover{color:rgb(0,0,0);background:transparent;border:0px solid gray;padding-left:25;background:rgb(240, 240, 240);}
QListWidget::Item:selected{border-image:url(images/listwidget_h.png); color:rgb(0,0,0);border:0px solid gray;}
QListWidget::Item:selected:active{background:rgb(229, 229, 229);color:#000000;border-width:0;}
</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="localListWidget">
    <widget class="myListWidget" name="localMusicList">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>1071</width>
       <height>431</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget{color:rgb(54,54,54); background:rgb(255,255,255);border:1px solid gray;outline:0px;}
QListWidget::Item{height:30px;border:0px solid gray;padding-left:50px}
QListWidget::Item:hover{background:transparent;border:0px solid gray;background:rgb(240, 240, 240); padding-left:55px}
QListWidget::Item:selected{border-image:url(images/listwidget_h.png);color:#000;border:0px solid gray;}
QListWidget::Item:selected:active{background:rgb(229, 229, 229);border-width:0;}
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addLocalMusicBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;添加本地音乐&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="firstWidget"/>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>580</y>
     <width>1281</width>
     <height>141</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="currentTime">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="prevSong">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>上一首（Ctrl+Left）</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="shortcut">
     <string>Ctrl+Left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pause">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>播放/暂停（Space）</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="shortcut">
     <string>Space</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="playMode">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>10</y>
      <width>241</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="random">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;随机播放&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="loop">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;列表循环&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="single">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;单曲循环&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="totalTime">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>90</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="mySlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>581</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::add-page:Horizontal{     
    background-color: rgba(205, 205, 205,100);
    height:4px;  
}
QSlider::sub-page:Horizontal{  
	background-color: rgb(255, 78, 78);
    height:1px;
 }
QSlider::groove:Horizontal{   
	background: transparent; 
    height:4px;
}  
QSlider::handle:Horizontal{ 
    height:16px; 
    width:16px; 
	image: url(:/image/pointL.png);
	margin: -6 0px;
}
QSlider::groove:Horizontal:hover{   
	background: transparent; 
    height:6px;

	
}  
QSlider::handle:Horizontal:hover{ 
    height:16px; 
    width:16px; 
	image: url(:/image/pointM.png);
	margin: -6 0px;
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="nextSong">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>下一首（Ctrl+Right）</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="shortcut">
     <string>Ctrl+Right</string>
    </property>
   </widget>
   <widget class="QLabel" name="songName">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>歌名</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="muteBtn">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>85</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="mySlider" name="volumeSilder">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>90</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::add-page:Horizontal{     
    background-color: rgba(205, 205, 205,100);
    height:4px;  
}
QSlider::sub-page:Horizontal{  
	background-color: rgb(255, 78, 78);
    height:1px;
 }
QSlider::groove:Horizontal{   
	background: transparent; 
    height:3px;
	border-radius: 4px;
}  
QSlider::handle:Horizontal{ 
    height:16px; 
    width:16px; 
	image: url(:/image/pointL.png);
	margin: -9 0px;
}
QSlider::groove:Horizontal:hover{   
	background: transparent; 
	border-radius: 4px;
    height:6px;

	
}  
QSlider::handle:Horizontal:hover{ 
    height:16px; 
    width:16px; 
	image: url(:/image/pointM.png);
	margin: -6 0px;
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="unMuteBtn">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>85</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>静音</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="albumPic">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>专辑图片</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/image/defaultAlbumPic.jpg</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>20</width>
     <height>501</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>572</y>
     <width>1281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>1281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame2{
background-color:rgb(102, 190, 252
);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lrcLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>851</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文中宋</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>歌词</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="logOutBtn">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>25</y>
      <width>50</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#logOutBtn{
	border: 0px;
	background: transparent;
}</string>
    </property>
    <property name="text">
     <string>注销</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginBtn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 0px;
}
#loginBtn{
    image: url(:/image/user.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="userNickName">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>myListWidget</class>
   <extends>QListWidget</extends>
   <header location="global">mylistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>mySlider</class>
   <extends>QSlider</extends>
   <header location="global">myslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
